explainer.md

```diff
@@ -1,121 +1,159 @@
+
+## Getting pending sync details
+
+As seen in the previous code examples, `sync.register()` and `syncEvent.registration` expose a sync registration object. You can also fetch them using `sync.getRegistration`, `sync.getRegistrations`, and `periodicSync.getRegistration`, `periodicSync.getRegistrations`.
+
+For example, to unregister a single one-off sync:
+
 ```js
-// Returned in order of registration.
-swRegistration.syncManager.getRegistrations().then(function(regs) {
-  for (reg of regs) reg.unregister();
+navigator.serviceWorker.ready.then(function(registration) {
+  registration.sync.getRegistration('outbox').then(function(syncReg) {
+    syncReg.unregister();
+  });
 });
 ```
 
+To unregister all periodic syncs, except "get-latest-news":
+
 ```js
-swRegistration.syncManager.getRegistration('weeklySync').then(function(reg) {
-  reg.unregister();
+navigator.serviceWorker.ready.then(function(registration) {
+  registration.periodicSync.getRegistrations().then(function(syncRegs) {
+    syncRegs.filter(function(reg) {
+      return reg.tag != 'get-latest-news';
+    }).forEach(function(reg) {
+      reg.unregister();
+    });
+  });
 });
 ```
 
 ## Checking for Permission
-If the origin doesn't have permission to use background sync then registration will fail. A prompt for permission can only occur from the page and not the service worker (which runs in the background). So call registration from the page first to invoke the permission request before using it in the service worker.
+
+Permissions for `sync` and `periodicSync` are entirely separate, and `periodicSync` is expected to be more difficult to obtain permission for.
 
 ```js
-swRegistration.syncManager.hasPermission().then(function(status) {
-  alert("Permission status: " + status);
+navigator.serviceWorker.ready.then(function(registration) {
+  registration.periodicSync.permissionState().then(function(state) {
+    if (state == 'prompt') showSyncRegisterUI();
+  });
 });
 ```
 
 ## Notes
 
-  * Since Service Workers are a requirement for Background Synchronization, and since Service Workers are limited to HTTPS origins, sites served without encryption will always fail to register for synchronization.
-  * All fetches during onsync events must be HTTPS. HTTP fetches will be rejected.
-  * Background Synchronization is not likely to be available to all web applications, not even all apps served over SSL. Browsers may choose to limit the set of applications which can register for synchronization based on quality signals that aren't a part of the visible API.
-  * `onsync` event handlers aren't allowed to run forever. Service workers may cap the total runtime of handlers, so it pays to try to batch work and count on needing to resume from failure. Also, test.
+* Since Service Workers are a requirement for sync, and since Service Workers are limited to HTTPS origins, that restriction applies here too.
+* All fetches during sync events must be HTTPS. HTTP fetches will be rejected.
```

Comment:
Will update when https://github.com/slightlyoff/ServiceWorker/issues/671 is resolved